opennms-base-assembly/src/main/filtered/etc/opennms.properties

```diff
@@ -1,622 +1,179 @@
-#
-# Default: false
+#org.opennms.snmp.dataCollectionConfig.reloadCheckInterval=30000
+org.opennms.rrd.storeByGroup=false
 org.opennms.rrd.storeByForeignSource=false
-
-# RRD base directory 
 rrd.base.dir=${install.share.dir}/rrd
-
-# RRD binary path (only used if the JniRrdStrategy is used)
 rrd.binary=${install.rrdtool.bin}
+#org.opennms.netmgt.collectd.SnmpCollector.forceRescan = false
+#org.opennms.netmgt.collectd.SnmpCollector.limitCollectionToInstances=false
 
-# Format rule used when setting the datasource value for threshold events
-# (check NMS-3473).
-# In order to properly use scientific notation for the datasource's value,
-# change the format rule to something like this '##.##E0' to avoid NMS-4126.
-#
-# Default: ##.##
-#org.opennms.threshd.value.decimalformat=##.##
-
-# Specifies the amount of time to wait (expressed in milliseconds) until the
-# reload container physically checks if the datacollection-config.xml file
-# has been changed.
-#
-# Default: 30 seconds
-#org.opennms.snmp.dataCollectionConfig.reloadCheckInterval=30000
-
-#
-# Collectd Instrumentation class
-# org.opennms.collectd.instrumentationClass=org.opennms.netmgt.collectd.DefaultCollectdInstrumentation
-
-# Enable an aggresive validation against the last modification time of the strings.properties files.
-# This is useful only if the OpenNMS WebUI is running on a different server.
-# Check NMS-5806 for more details.
-#
-# Default: false
-#org.opennms.utils.propertiesCache.enableCheckFileModified=false
-
+### Trouble Ticketing
 
-###### TROUBLE TICKETING ######
-# The ticketer responsible for creating tickets from the Alarm details and passing these 
-# to the ticket plugin.
 opennms.ticketer.servicelayer=org.opennms.netmgt.ticketd.DefaultTicketerServiceLayer
-
-# The ticket plugin implementation to use to create tickets via the Alarm details
-# pages (if configured).  The NullTicketerPlugin just does nothing when attempting
-# to create tickets.
-# NOTE: if you enable a trouble-ticketing plugin here, you must also edit
-# $OPENNMS_HOME/jetty-webapps/opennms/WEB-INF/configuration.properties
 opennms.ticketer.plugin=org.opennms.netmgt.ticketd.NullTicketerPlugin
-
-# Comment out the NullTicketerPlugin line above and uncomment the lines below
-# to enable OtrsTicketerPlugin
-# You will need to adjust alarmTroubleTicketLinkTemplate to suit your OTRS Base URL
-# Note that you will need to install the OpenNMS module for OTRS in your OTRS server
-
 #opennms.ticketer.plugin=org.opennms.netmgt.ticketer.otrs.OtrsTicketerPlugin
 #opennms.alarmTroubleTicketEnabled = true
 #opennms.alarmTroubleTicketLinkTemplate = <a href="http://localhost/otrs/index.pl?Action=AgentTicketZoom&TicketNumber=\${id}">\${id}</a>
-
-# Comment out the NullTicketerPlugin line above and uncomment the lines below
-# to enable RtTicketerPlugin
-# You will need to adjust alarmTroubleTicketLinkTemplate to suit your RT Base URL
-
 #opennms.ticketer.plugin=org.opennms.netmgt.ticketer.rt.RtTicketerPlugin
```

Comment:
This section needs a one-line explanation that all of these properties are used to enable RT ticketing.